src/components/comments.js

```diff
@@ -34,6 +37,29 @@ const userRef = {
     },[]);
 
 
+    const handleSubmit = (event) => {
+        event.preventDefault()
+        setIsLoading(true);
+        setErr(null)
+        fetch(`https://hannybees-news-app.herokuapp.com/api/articles${articleIdRoute}/comments`, {
+    method: 'POST',
+    body: JSON.stringify({
+    username:'cooljmessy',
+    body: `${newComment}`,
```

Comment:
I dont think this needs to be inside a template literal, but its no big issue :)